src/main/java/co/cfly/faces/components/CopyComponent.java

```diff
@@ -30,10 +30,9 @@ public void encodeBegin(FacesContext context) throws IOException {
             }
 
             ResponseWriter writer = context.getResponseWriter();
-            writer.startElement("button", this);
-            writer.writeAttribute("type", "button", "type");
+            writer.startElement("a", this);
             writeId(context);
-            String styleClass = (String) getAttributes().getOrDefault("styleClass", "btn btn-outline-secondary btn-sm");
+            String styleClass = (String) getAttributes().getOrDefault("styleClass", "link");
             String styleClassValue = RendererTools.spaceSeperateStrings("clipboard-copy-button", styleClass);
```

Comment:
this is no longer a button so the css class used to match should not be "button" anymore

clipboard-copy-element would make more sense.

This has to be updated in portal core.js as that is looking for the css class to enable